components/cas-inbound-authenticator/src/main/java/org/wso2/carbon/identity/sso/cas/constants/CASConstants.java

```diff
@@ -21,6 +21,7 @@
     public static final String CAS_CONFIG_NAME = "cas";
     public static final String CAS_FRIENDLY_NAME = "CAS Configuration";
     public static final String CAS_APP_TYPE = "hidden";
+    public static final String MULTI_ATTRIBUTE_SEPARATOR = "MultiAttributeSeparator";
```

Comment:
Isn't this avaialble in somewhere else ? This constant should be available

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-cas/pull/21/commits/3f5b29c4b26a91f1fdb208e6703dd96d6477e11d